### DISCUSSION TITLE
Ability to type only quotes from a specific source

### DISCUSSION THREAD

**Discussion by Jonny-exe** [monkeytypegame/monkeytype#3675] [Ideas]
Hi, I would like to suggest making it possible to search and type quotes from a specific source. 
I could imagine this being in the funbox section. 

For example: 
If you searched "Lord of the Rings", you would get all the quotes with "Lord of the Rings" as their source in a loop.
If this sounds good I would love to do it myself.

**Comment by vdawg-git** [comment]
Yes, I came here to propose a LoTR mode :D
This would be so nice
